src/kits/tracker/PoseView.cpp

```diff
@@ -35,6 +35,7 @@ All rights reserved.
 
 #include "PoseView.h"
 
+#include <iostream>
```

Comment:
What are you using form iostream?

Reply:
I removed all the debug output and I seem to have forgotten to remove the header file from being included. I fixed this in the latest commit.